src/components/meeting/meetingTime/JitsiComponentWrapper.js

```diff
@@ -33,23 +51,193 @@ export const JitsiWrapper = (props) => {
         setJitsiWidth('20%')
         setActionComponent(
             <div style={{float: "left", width: '80%', height: '80vh', overflow: 'scroll'}}>
-                <TaskWrapper id={16} submitLeftOffset={'25%'}/>
+                <TaskWrapper fetchTasks={getTasksByInterviewUUID} submitLeftOffset={'25%'}/>
             </div>
         )
     }
 
-    const getWhiteboardCode = () => {
-        if (props.roomName.length < 15) return 'abcdefghijklmnopqrst'
-        return props.roomName
-    }
+    const getTasksByInterviewUUID = () => tasksApi.getTasksByInterviewUUID(interviewId)
 
     return (
         <div>
+            {
+                admin &&
+                    <AdminMeetingDrawer interviewId={interviewId} companyId={companyId} />
+            }
             <MeetingFab onWhiteboard={addWhiteboard} onDefaultView={defaultView} onTask={showTask}/>
             <div style={{height: jitsiHeight, width: jitsiWidth, float: "left"}}>
-                <JitsiComponent admin={props.admin} roomName={props.roomName} displayName={props.displayName}/>
+                <JitsiComponent admin={admin} roomName={roomName} displayName={displayName}/>
             </div>
             {actionComponent}
         </div>
     )
+}
+
+
+const drawerWidth = 600;
+const useStyles = makeStyles((theme) => ({
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+  },
+  drawerOpen: {
+    width: drawerWidth,
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  drawerClose: {
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    overflowX: 'hidden',
+    width: theme.spacing(7) + 1,
+    display: "flex",
+    alignItems: "center",
+    [theme.breakpoints.up('sm')]: {
+      width: theme.spacing(9) + 1,
+    },
+  }
+}));
+
+const AdminMeetingDrawer = ({ interviewId, companyId }) => {
+
+    const [openNotesModal, setOpenNotesModal] = useState(false)
+    const [reload, setReload] = useState(false)
+    const [openTasksModal, setOpenTasksModal] = useState(false)
+
+    const { get, set } = useDevPassword()
```

Comment:
Yyyy to nie jest chyba najlepszy dobór nazw zmiennych XD

Reply:
Najlepiej byłoby zmienić sygnatury tych metod, które są w hooku